spacy_curated_transformers/cli/fill_config_transformer.py

```diff
@@ -0,0 +1,470 @@
+import json
+import sys
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, Optional, Tuple
+
+from confection import Config
+from spacy import util
+from spacy.cli._util import import_code, init_cli, parse_config_overrides
+from spacy.cli.init_config import save_config
+from typer import Argument as Arg
+from typer import Context
+from typer import Option as Opt
+from wasabi import Printer
+
+from .._compat import has_hf_transformers, has_huggingface_hub
+
+
+@init_cli.command(
+    "fill-curated-transformer",
+    context_settings={"allow_extra_args": True, "ignore_unknown_options": False},
+)
+def init_fill_curated_transformer_cli(
+    # fmt: off
+    ctx: Context,  # This is only used to read additional arguments
+    base_path: Path = Arg(..., help="Path to the base config file to fill", exists=True, allow_dash=True, dir_okay=False),
+    output_path: Path = Arg("-", help="Path to output .cfg file or '-' for stdout (default: stdout)", exists=False, allow_dash=True),
+    model_name: Optional[str] = Opt(None, "--model-name", "-m", help="Name of the Hugging Face model. If not provided, the model name will be read in from the encoder loader config"),
+    model_revision: Optional[str] = Opt(None, "--model-revision", "-r", help="Revision of the Hugging Face model (default: 'main')."),
+    transformer_name: Optional[str] = Opt(None, "--pipe-name", "-n", help="Name of the transformer pipe whose config is to be filled (default: first transformer pipe)."),
+    code_path: Optional[Path] = Opt(None, "--code-path", "--code", "-c", help="Path to Python file with additional code (registered functions) to be imported"),
+    # fmt: on
+):
+    """
+    Fetches the hyperparameters of a Curated Transformer model
+    from the Hugging Face Hub and fills in the config with them.
+
+    DOCS: https://spacy.io/api/cli#init-fill-curated-transformer
```

Comment:
Is this PR already created?

Reply:
This should part of the spaCy curated transformers docs PR - https://github.com/explosion/spaCy/pull/12677